doc/operator-notes.md

```diff
@@ -103,3 +103,23 @@ Specifying `size` as 0 means the partition should span to the end of the largest
 ### Unspecified partition size
 If `size` is not specified and a partition with the same number exists, it will use the value of the existing partition, unless wipePartitionEntry is set.
 If `size` is not specified and there is no existing partition, or wipePartitionEntry is set, `size` act as if it were set to 0 and use the size of the largest block.
+
+## Config Merging
+
+Ignition supports fetching, then merging multiple configs. This replaces the `append` functionality of the Ignition 2.x.0 specification. There are several rules that determine how configs get merged. When a child config is merged with a parent, generally the child config's values override the parent config's values.
+
+### Child configs take precedence when specified
+
+If a parent and child object are being merged, the fields in the child object take precedence over the fields in the parent stuct. If a field in the child object is not specified, the field from the parent is used instead.
```

Comment:
struct

Reply:
actually should be `config`. We shouldn't expose golang-isms here